dev/com.ibm.ws.jaxws.2.2.webcontainer_fat/fat/src/com/ibm/ws/jaxws/fat/LibertyCXFPropertiesTest.java

```diff
@@ -0,0 +1,104 @@
+/*******************************************************************************
+ * Copyright (c) 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.jaxws.fat;
+
+import static org.junit.Assert.assertNotNull;
+
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+import javax.net.ssl.HttpsURLConnection;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.ShrinkHelper;
+import com.ibm.websphere.simplicity.log.Log;
+
+import componenttest.annotation.Server;
+import componenttest.annotation.SkipForRepeat;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.topology.impl.LibertyServer;
+import componenttest.topology.utils.HttpUtils;
+
+/*
+ * Tests the enablement of cxf.multipart.attachment
+ * and cxf.ignore.unsupported.policy property migrations
+ * from CXF 2.6.2. Test designed to work for EE9
+ *
+ * This test is using WebServiceRefFeaturesTest's infrastructure
+ * with additional codes and wsdls tailored for this test only
+ */
+@RunWith(FATRunner.class)
+@SkipForRepeat(SkipForRepeat.NO_MODIFICATION)
+public class LibertyCXFPropertiesTest {
+
+    @Server("PropertyTestServer")
```

Comment:
I'd change the name of the server to `LibertyCXFPropertiesServer` to continue one the better practices of our test buckets that match the server name with the fat test name, making the test easier to find when debugging failures.